packages/serverless-components/nextjs-cdk-construct/src/utils/readInvalidationPathsFromManifest.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from "path";
+import { OriginRequestDefaultHandlerManifest } from "@sls-next/lambda-at-edge";
+
+const dynamicPathToInvalidationPath = (dynamicPath: string) => {
+  const [firstSegment] = dynamicPath.split("/:");
+  return path.join(firstSegment || "/", "*");
+};
+
+export const readInvalidationPathsFromManifest = (
+  manifest: OriginRequestDefaultHandlerManifest
+): string[] => {
+  return [
+    ...Object.keys(manifest.pages.html.dynamic).map(
+      dynamicPathToInvalidationPath
+    ),
+    ...Object.keys(manifest.pages.html.nonDynamic),
+    ...Object.keys(manifest.pages.ssr.dynamic).map(
+      dynamicPathToInvalidationPath
+    ),
+    ...Object.keys(manifest.pages.ssr.nonDynamic),
+    ...Object.keys(manifest.pages.ssg?.dynamic || {}).map(
+      dynamicPathToInvalidationPath
+    ),
+    ...Object.keys(manifest.pages.ssg?.nonDynamic || {})
+  ];
```

Comment:
Invalidate all known nonDynami paths, and invalidate all dynamic paths using a wildcard i.e. `/users/*`

Reply:
Can user still invalidate all paths or custom set of paths? I think because if there are many individual invalidations the costs would add up eventually.

Reply:
Good catch, Ill add an option for this. I also think that this function should probably filter out any subpaths of wildcards, as currently it would still create an invalidation for `/users/*` and `/users/some/path` if `/users/:id` and `/users/some/path` exist... However `/users/some/path` would actually be invalidated by `/users/*`.

Reply:
@dphang, I've just put in a fix for this here: https://github.com/serverless-nextjs/serverless-next.js/pull/878/commits/7aeaec92c91f42f051b28c31863cb3cbe75e9966